### PR TITLE
Added: alt attribute to few images

### PR DESCRIPTION
**Pull request by chaitanyaj2222**
#### Summary

* I have added alt attributes to a handful of items in the content
#### Motivation

* Adding the alt attribute to the image displays the value of `alt` attribute when the respective `image fails to load` rather than leaving the space blank. It is also read aloud by the screen readers when there is an image to describe the user about the image.
#### Supporting details

@estelle  has collected the list of images from the documentation that do not have the alt attribute. [link to the spreadsheet](https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0)
#### Metadata

- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Chaitanya7666** [comment]
#19334

**Review by bsmth — CHANGES_REQUESTED** [review]
Thanks a lot for the submissions. The alt text is too long. Could you please shorten each alt text to __around 100-125 characters__, please? Ideally the alt text is a single sentence that summarizes the content and is quick and easy to understand.

Thank you!

File: files/en-us/web/api/canvas_api/tutorial/drawing_shapes/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Now that we have set up our [canvas environment](/en-US/docs/Web/API/Canvas_API/
 
 Before we can start drawing, we need to talk about the canvas grid or **coordinate space**. Our HTML skeleton from the previous page had a canvas element 150 pixels wide and 150 pixels high.
 
-![](canvas_default_grid.png)
+![Canvas grid: center is at top left corner. Each tiny square corresponds to 1 pixel on the canvas. A diagram of a square is at 'x' units from Y-axis and 'y' units from X-axis and it is filled in royal blue color. The height of the diagram is taken along Y-axis and the width of the diagram is taken along X-axis.](canvas_default_grid.png)
```
Comment (estelle) [inline files/en-us/web/api/canvas_api/tutorial/drawing_shapes/index.md:21]:
```suggestion
![Canvas grid with a blue square demonstrating coordinates and axes.](canvas_default_grid.png)
```
The paragraph following the image has all the details about this image. The alt attribute should therefore included what the image conveys without being redundant with the content.

File: files/en-us/web/api/canvas_api/tutorial/drawing_shapes/index.md [hunk 2]
```diff
@@ -301,7 +301,7 @@ The next type of paths available are [Bézier curves](/en-US/docs/Glossary/Bezie
   - : Draws a cubic Bézier curve from the current pen position to the end point specified by `x` and `y`, using the control points specified by (`cp1x`, `cp1y`) and (cp2x, cp2y).
 
 The difference between these is that a quadratic Bézier curve has a start and an end point (blue dots) and just one **control point** (indicated by the red dot) while a cubic Bézier curve uses two control points.
-![](canvas_curves.png)
+![Quadratic curve and Bezier curves: Quadractic curve has a start and an end point and one control point while a Bezier curve uses 2 control points. The start and end points are indicated by the blue dots while the control points are indicated by red dots.](canvas_curves.png)
```
Comment (estelle) [inline files/en-us/web/api/canvas_api/tutorial/drawing_shapes/index.md:304]:
```suggestion
![Quadratic and Bezier curve comparison.](canvas_curves.png)
```

No need to duplicate content that is in the text: screen reader users don't want to hear the information twice. This page's images are really well described in the text, which is rare.

File: files/en-us/web/api/canvas_api/tutorial/transformations/index.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ The first of the transformation methods we'll look at is `translate()`. This met
 - {{domxref("CanvasRenderingContext2D.translate", "translate(x, y)")}}
   - : Moves the canvas and its origin on the grid. `x` indicates the horizontal distance to move, and `y` indicates how far to move the grid vertically.
 
-![](canvas_grid_translate.png)
+![A canvas grid with center at top left corner. The canvas and it's origin are moved on the grid by 'x' units horizontally and 'y' units vertically.](canvas_grid_translate.png)
```
Comment (estelle) [inline files/en-us/web/api/canvas_api/tutorial/transformations/index.md:84]:
```suggestion
![The canvas is pushed down and to the right, or translated, from it's origin point on the grid by 'x' units horizontally and 'y' units vertically.](canvas_grid_translate.png)
```

File: files/en-us/web/api/canvas_api/tutorial/transformations/index.md [hunk 4]
```diff
@@ -123,7 +123,7 @@ The second transformation method is `rotate()`. We use it to rotate the canvas a
 - {{domxref("CanvasRenderingContext2D.rotate", "rotate(angle)")}}
   - : Rotates the canvas clockwise around the current origin by the `angle` number of radians.
 
-![](canvas_grid_rotate.png)
+![Canvas grid with center at top left corner: The grid is rotated around the origin in clock-wise direction by alpha radians.](canvas_grid_rotate.png)
```
Comment (estelle) [inline files/en-us/web/api/canvas_api/tutorial/transformations/index.md:126]:
```suggestion
![The default origin point is at the top left, 0 degrees is horizontal and to the right. The rotation point starts from the origin point and goes clockwise.](canvas_grid_rotate.png)
```

File: files/en-us/web/api/canvas_api/tutorial/using_images/index.md [hunk 5]
```diff
@@ -203,7 +203,7 @@ The third and last variant of the `drawImage()` method has eight parameters in a
 
 To really understand what this does, it may help to look at this image:
 
-![](canvas_drawimage.jpg)
+![There is the source image(rhinoceros in water) on the top left corner. It is rectangular and it's top left corner is at co-ordinates (sx,sy) and whose width and height are sWidth and sHeight respectively. The translated image is shown at the bottom right corner. The top-left corner of the image in the destination canvas is at co-ordinates (dx,dy) and it is scaled to the size specified by dWidth and dHeight.](canvas_drawimage.jpg)
```
Comment (estelle) [inline files/en-us/web/api/canvas_api/tutorial/using_images/index.md:206]:
```suggestion
![The source image is rectangular with top left coordinates are sx and sy and the width and height are sWidth and sHeight respectively. That is translated to the destination canvas where the top-left corner is at coordinates dx and dy, maintaining its aspect ration with a width and height of dWidth and dHeight respectively.](canvas_drawimage.jpg)
```

**Review by estelle — COMMENTED** [review]
Thank you. I made several suggestions and a few comments. When writing alternative text you don't need to describe what is in the image. Rather, you want to relay the information that the image is trying to convey if that information is not in the surrounding text.

**Comment by Chaitanya7666** [comment]
Thanks for reviewing the PR @bsmth and @estelle . 

@estelle , Is it okay to describe what's in the image if the surrounding text does not have any information about the image?

**Review by bsmth — APPROVED** [review]
LGTM, thanks for the additions 🙌🏻

**Pull request merged**